man-roxygen/pheight.R

```diff
@@ -1,5 +1,5 @@
 #' @param pwidth Default width of plots printed to files in units of
```

Comment:
Lines 1-2 should be removed, right?

Reply:
Of course - sorry for a such a silly mistake!